### PR TITLE
Feature uploading image

### PR DESCRIPTION
**Pull request by petyaNikleva**

### CODE REVIEW

File: pages/user-profile/user-profile.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import authService from "../../services/authenticationService.js";
+
+export function currentUser () {
+    let email = authService.getLoggedUser()
```
Comment (NastyaSinkevich) [inline pages/user-profile/user-profile.js]: use const there

File: pages/user-profile/user-profile.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    let user = authService.getUser(email);
```
Comment (NastyaSinkevich) [inline pages/user-profile/user-profile.js]: just `return authService.getUser(email)`
you don't need variables there.

File: services/authenticationService.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ class userService {
     constructor() {
     }
 
-    checkIsLoggedUser() {
+    getLoggedUser() {
         let loggedUser = localStorage.getItem('loggedUser');
```
Comment (NastyaSinkevich) [inline services/authenticationService.js]: `return localStorage.getItem('loggedUser');`

File: model/User.js [hunk 4]
```diff
@@ -1,9 +1,10 @@
 export class User {
-    constructor(email, firstName, lastName, dateOfBirth, password) {
+    constructor(email, firstName, lastName, dateOfBirth, password, image) {
         this.email = email;
         this.firstName = firstName;
         this.lastName = lastName;
         this.dateOfBirth = dateOfBirth;
         this.password = password;
+        this.image = image;
     }
```
Comment (NastyaSinkevich) [inline model/User.js:9]: Also in the user class, I think that you can add user methods.

File: services/authenticationService.js [hunk 5]
```diff
@@ -57,10 +57,23 @@ class userService {
         localStorage.setItem('users', JSON.stringify(users));
     }
 
+    // checkIsUploadedPicture(email) {
+    //     let user = this.getUser(email);
+    //     console.log(user);
+    // }
+
+    uploadImage(email, imgId) {
+        let user = this.getUser(email);
```
Comment (NastyaSinkevich) [inline services/authenticationService.js]: use const

File: services/authenticationService.js [hunk 6]
```diff
@@ -2,7 +2,7 @@ class userService {
     constructor() {
```
Comment (NastyaSinkevich) [inline services/authenticationService.js:2]: classes prefer to be named with the uppercase letter `UserService`

File: services/authenticationService.js (hunk 5 above)
Comment (NastyaSinkevich) [reply services/authenticationService.js]: https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/

read about const and how it works with objects and arrays

File: server/server.js [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+const path = require('path');
+const cors = require('cors');
+
+const app = express();
+
+app.use(cors());
+
+const fileStorageEngine = multer.diskStorage({
+    destination: (req, file, cb) => {
+        cb(null, '../uploads'); // it save image in filder which is outside the project
```
Comment (NastyaSinkevich) [inline server/server.js]: it is okay

Comment (petyaNikleva) [reply server/server.js]: Ok. thanks for the answer.

File: services/authenticationService.js (hunk 6 above)
Comment (petyaNikleva) [reply services/authenticationService.js:2]: Yes, I forgot about it when created the file. I've changed it.

File: services/authenticationService.js (hunk 5 above)
Comment (petyaNikleva) [reply services/authenticationService.js]: Thanks. Corrected.

File: services/authenticationService.js (hunk 3 above)
Comment (petyaNikleva) [reply services/authenticationService.js]: Corrected

File: pages/user-profile/user-profile.js (hunk 2 above)
Comment (petyaNikleva) [reply pages/user-profile/user-profile.js]: Corrected.

File: pages/user-profile/user-profile.js (hunk 1 above)
Comment (petyaNikleva) [reply pages/user-profile/user-profile.js]: Corrected.

**Pull request merged**